compiler/src/iree/compiler/Codegen/Transforms/Transforms.h

```diff
@@ -52,7 +52,8 @@ template <typename AllocLikeOpType>
 std::optional<Value> hoistOneStaticallyBoundAllocation(
     mlir::FunctionOpInterface funcOp, OpBuilder &builder, Location loc,
     MemRefType allocaType, ValueRange dynamicSizes,
-    std::optional<uint64_t> alignment);
+    std::optional<uint64_t> alignment,
+    std::optional<VscaleRange> vscaleRange = {});
```

Comment:
nit: it looks better if the default value is `std::nullopt`, but not `{}`. Same for below changes